src/main/java/com/cozyhome/onlineshop/userservice/security/JWT/JwtAuthEntryPoint.java

```diff
@@ -1,24 +1,29 @@
 package com.cozyhome.onlineshop.userservice.security.JWT;
 
-import java.io.IOException;
-
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.security.core.AuthenticationException;
 import org.springframework.security.web.AuthenticationEntryPoint;
 import org.springframework.stereotype.Component;
 
-import jakarta.servlet.ServletException;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.servlet.HandlerExceptionResolver;
 
 @Slf4j
 @Component
 public class JwtAuthEntryPoint implements AuthenticationEntryPoint{
 
-	@Override
+    @Autowired
+    @Qualifier("handlerExceptionResolver")
+    private HandlerExceptionResolver resolver;
+
+    @Override
     public void commence(HttpServletRequest request, HttpServletResponse response,
-            AuthenticationException authException) throws IOException, ServletException {
+                         AuthenticationException authException) {
```

Comment:
this method will not throws IOException, ServletException? or why they was deleted?

Reply:
yes, with the current implementation of the method they are not active